app/screens/get-started-screen/use-device-token.ts

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from "react"
+import appCheck from "@react-native-firebase/app-check"
+import Config from "react-native-config"
+
+const rnfbProvider = appCheck().newReactNativeFirebaseAppCheckProvider()
+rnfbProvider.configure({
+  android: {
+    provider: __DEV__ ? "debug" : "playIntegrity",
+    debugToken: Config.APP_CHECK_ANDROID_DEBUG_TOKEN,
+  },
+  apple: {
+    provider: __DEV__ ? "debug" : "appAttestWithDeviceCheckFallback",
+    debugToken: Config.APP_CHECK_IOS_DEBUG_TOKEN,
+  },
+})
+
+appCheck().initializeAppCheck({ provider: rnfbProvider, isTokenAutoRefreshEnabled: true })
+
+export const getDeviceToken = async (): Promise<string | undefined> => {
+  try {
+    const result = await appCheck().getToken()
+    const token = result.token
+    return token
+  } catch (err) {
+    console.log("getDeviceToken error", err)
+    return undefined
+  }
+}
+
+const useDeviceToken = ({
```

Comment:
useDeviceToken is confusing to me. 

we also use deviceToken both in the context of the the backend and frontend for the device notification tokens. 

I think maybe renaming this to useDeviceAccount() or something like that would help not to confuse both concepts.

Reply:
I think we can switch it to `useAppCheckToken`